animations/test/misc/animated_list_test.dart

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:math';
+import 'package:animations/src/misc/animated_list.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+Widget createAnimatedListDemoScreen() => MaterialApp(
+      home: AnimatedListDemo(),
+    );
+
+void main() {
+  group('Animated List Tests', () {
+    testWidgets('Initial Length of List', (tester) async {
+      await tester.pumpWidget(createAnimatedListDemoScreen());
+
+      // Get the initial Length of List.
+      var initialLength = tester.widgetList(find.byType(ListTile)).length;
+
+      // initial length of List should be equal to 5.
```

Comment:
nit: The comment should start with capital letter. There are few other comments too which don't start with capital letter.

Reply:
Noted.  Thanks for review.